Source/KakapoServer.swift

```diff
@@ -37,15 +39,20 @@ import Foundation
  ```
  */
 public final class KakapoServer: NSURLProtocol {
-    
+
+    /**
+     The overall list of `Router` objects, which is know to all `KakapoServer` instances.
+     Visibility is on the `KakapoServer` class level, because `NSURLProtocol` just allows us to register
+     class elements (no instances are possible).
+     */
```

Comment:
No need to add documentation here since it's a private var -> implementation detail on how the Router and Server communicate and Server keeps storage of routers, nothing that the client needs to be aware of.